oodeel/methods/base.py

```diff
@@ -129,7 +115,18 @@ def fit(
             else:
                 self.compute_react_threshold(model, fit_dataset)
 
-        self.feature_extractor = self._load_feature_extractor(model)
+        if (feature_layers_id == []) and (self.requires_internal_features):
+            raise ValueError(
+                "Explicitely specify output_layers_id=[layer0, layer1,...], "
```

Comment:
Typo: `feature_layers_id` instead of `output_layers_id`
BTW, I think that the correct spelling is "explicitly" (without E). However, I'm not 100% sure